include/sway/server.h

```diff
@@ -157,6 +157,7 @@ extern struct sway_debug debug;
 
 extern bool allow_unsupported_gpu;
 
+void sway_terminate(int exit_code);
```

Comment:
We should be able to get rid of the forward decl in `sway/commands/exit.c` now.

Reply:
adding this declaration is no longer part of this series